packages/core/src/account/base.ts

```diff
@@ -379,7 +380,7 @@ export abstract class BaseSmartContractAccount<
       );
     }
 
-    return trim(storage);
+    return toHex(trim(storage));
```

Comment:
this is a bug I think. I ran into this issue somewhere else. This needs to just be `trim` and not `toHexed` again. The issue I ran into was that if you `toHex` the hex string it generates a new `hex` which is the hex representation of the bytes of the previous hex string